home/templates/home/index.html

```diff
@@ -98,35 +98,35 @@
                 <div class="carousel-caption">
                     <div class="col-md-12 col-xs-12 text-center">
                       <h2>KOSS</h2>
-                      <h3 class="animated2"> <b>Creative</b> Modern and clean </h3>
+                      <h3> Kharagpur <b>Open Source</b> Society </h3>
                       <div class="line"></div>
-                      <p class="animated3">Unique clean design</p>
+                      <p class="animated3">CODE | DEBUG | CODE</p>
```

Comment:
FOSS is a lot more than coding and debugging.
 
**CODE | TEST | DEBUG (if possible | DOCUMENT)**  looks better to me